third_party/transducer/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.5)
+
+PROJECT(rnnt_release)
+
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -O2")
```

Comment:
I wonder if this flag should be hardcoded here, and wonder why it's `-O2`, instead of `-O3`.